include/ceed-backend.h

```diff
@@ -1,3 +1,3 @@
 #include <ceed/backend.h>
 
-#warning <ceed-backend.h> is deprecated and will be removed in libCEED v0.9; use <ceed/backend.h>
+#warning "<ceed-backend.h> is deprecated and will be removed in libCEED v0.9; use <ceed/backend.h>"
```

Comment:
We already release v0.9....

Reply:
Yeah. That was an out of cycle release, so perhaps we delete these files now in anticipation of v0.10?

Reply:
Harmless, but I'm fine either way.